Conference-Android/app/src/main/java/com/sagetech/conference_android/app/ui/activities/ConferenceSessionListActivity.java

```diff
@@ -0,0 +1,168 @@
+import android.support.v4.app.NavUtils;
+import android.support.v7.app.ActionBar;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import com.sagetech.conference_android.app.R;
+import com.sagetech.conference_android.app.api.ConferenceController;
+import com.sagetech.conference_android.app.model.ConferenceSessionData;
+import com.sagetech.conference_android.app.ui.listener.RecyclerItemClickListener;
+import com.sagetech.conference_android.app.ui.presenter.ConferenceSessionListActivityPresenter;
+import com.sagetech.conference_android.app.ui.presenter.IConferenceSessionActivity;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+import java.util.Locale;
+
+import javax.inject.Inject;
+
+import butterknife.ButterKnife;
+import butterknife.InjectView;
+import timber.log.Timber;
+
+/**
+ * Created by adam on 2/21/15.
+ */
+public class ConferenceSessionListActivity extends InjectableActionBarActivity implements IConferenceSessionActivity {
+    @Inject
+    ConferenceController conferenceController;
+
+    @InjectView(R.id.confView)
+    RecyclerView mRecyclerView;
+
+    private RecyclerView.Adapter mAdapter;
+    private RecyclerView.LayoutManager mLayoutManager;
+    private ConferenceSessionListActivityPresenter presenter = null;
+    private Integer conferenceId;
+
+    @Override
+    public void populateConferenceSessions(List<ConferenceSessionData> datas) {
+        mAdapter = new ConferenceSessionsAdapter(datas);
+        mRecyclerView.setAdapter(mAdapter);
+    }
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_conferences);
+
+        final ActionBar actionBar = getSupportActionBar();
+        actionBar.setTitle(R.string.app_name);
+        actionBar.setDisplayHomeAsUpEnabled(true);
+
+        ButterKnife.inject(this);
+
+        mRecyclerView.setHasFixedSize(true);
+
+        mLayoutManager = new LinearLayoutManager(this);
+        mRecyclerView.setLayoutManager(mLayoutManager);
+
+        mRecyclerView.addOnItemTouchListener(
+                new RecyclerItemClickListener(this, new RecyclerItemClickListener.OnItemClickListener() {
+                    @Override public void onItemClick(View view, int position) {
+
+                        Timber.d(String.format("View Clicked: %s", view.getTag()));
+
+                        Intent eventDetailIntent = new Intent(view.getContext(), EventDetailActivity.class);
+                        eventDetailIntent.putExtra("id", (Long)view.getTag());
+                        startActivity(eventDetailIntent);
+                    }
+                })
+        );
+
+        conferenceId = getIntent().getExtras().getInt("id");
```

Comment:
I was going to ask you about that tomorrow.  I noticed inconsistencies between using Longs/Integers in the models and api parameters.  I was going to propose we use Longs for all ids.  If you're cool with that, I'll make the changes throughout.